broker-cli/cmd/instances.go

```diff
@@ -19,23 +19,22 @@ import (
 	"log"
 
 	"github.com/GoogleCloudPlatform/k8s-service-catalog/broker-cli/client/adapter"
-	"github.com/GoogleCloudPlatform/k8s-service-catalog/broker-cli/client/osb"
 	"github.com/GoogleCloudPlatform/k8s-service-catalog/broker-cli/cmd/flags"
 	"github.com/spf13/cobra"
 )
 
-// Instance is a Service Instance.
-type Instance struct {
-	ID            string `json:"instance_id"`
-	ServiceID     string `json:"service_id"`
-	PlanID        string `json:"plan_id"`
-	CreateTime    string `json:"createTime"`
-	NumOfBindings int    `json:"numOfBindings"`
+// instance is a Service Instance.
+type instance struct {
+	ID            string
+	serviceID     string
+	planID        string
+	createTime    string
+	arrOfBindings []string
```

Comment:
simple var names! `bindings` would be sufficient.

Reply:
done